### PR TITLE
JAMES-3302 Fix a dead link

### PR DESCRIPTION
**Pull request by chibenwa**
This was modified from https://james.staged.apache.org/james-project/3.6.0/servers/distributed.html
using the [edit this page] button <3

### CODE REVIEW

**Review by Arsnael — APPROVED** [review]

**Comment by jeantil** [comment]
I can't wait to see one of the pending PR merged to master to see if the website is indeed automatically rebuilt and republished :)

File: docs/modules/servers/pages/distributed/configure/elasticsearch.adoc [hunk 1]
```diff
@@ -111,6 +111,9 @@ xref:distributed/operate/webadmin.adoc#reindexing_all_mails[a full re-indexing]n
 
 == Exporting metrics directly to ElasticSearch
 
+WARNING: A separate ElasticSearch server is required, as an ElasticSearch 6.3 is needed, which is
+not compatible with ELasticSearch 7.10, required by James.
```
Comment (Arsnael) [inline docs/modules/servers/pages/distributed/configure/elasticsearch.adoc:115]:
```suggestion
not compatible with ElasticSearch 7.10, required by James.
```

**Comment by Arsnael** [comment]
> I can't wait to see one of the pending PR merged to master to see if the website is indeed automatically rebuilt and republished :)

We have an automatic rebuild for the website now? I didn't know... Hope it works :)

**Comment by chibenwa** [comment]
> We have an automatic rebuild for the website now? I didn't know... Hope it works :)

Jean did fight with the Jenkins :-)

**Review by LanKhuat — APPROVED** [review]

**Pull request merged**